File: API/Models/GameResultType.cs
namespace API.Models {
    public enum GameResultType {
        WonByResignation = 0,
        WonOnTime,
        WonByCheckmate,
        WonByAbandonment,
        WonByKingOfTheHill,
        WonByThreeCheck,
        WonByBugHouse,
        DrawByAgreement,
        DrawByStalemate,
        DrawByRepitition,
        DrawByInsufficientMaterial,
        DrawByTimeoutVsInsufficientMaterial,
        DrawBy50Move,
        LostByResignation,
        LostOnTime,
        LostByCheckmate,
        LostByAbandonment,
        LostByThreeCheck,
        LostByKingOfTheHill,
        LostByBugHousePartnerLose
    };
}

File: SusChessPlayers/src/app/models/game-result-type.ts
export const enum GameResultType
{
    WonByResignation = 'resigned',
    WonOnTime = 'timeout',
    WonByCheckmate = 'checkmated',
    WonByAbandonment = 'abandoned',
    DrawByAgreement = 'agreed',
    DrawByStalemate = 'stalemate',
    DrawByRepitition = 'repetition',
    DrawByInsufficientMaterial = 'insufficient',
    DrawByTimeoutVsInsufficientMaterial = 'timevsinsufficient',
    DrawBy50Move = '50move',
    LostByResignation = 'resigned',
    LostOnTime = 'timeout',
    LostByCheckmate = 'checkmated',
    LostByAbandonment = 'abandoned'
};